valhalla/jawn/src/middleware/auth.ts

```diff
@@ -19,37 +20,16 @@ export const logInPostHog = (
   const start = Date.now();
   const postHogClient = newPostHogClient();
   postHogClient?.capture({
-    distinctId: authParams?.organizationId ?? "unknown",
```

Comment:
you can leave this part - we don't do anything transactional here so you can go nuts